volatility3/framework/plugins/linux/pslist.py

```diff
@@ -49,6 +58,18 @@ def filter_func(x):
             return lambda _: False
 
     def _generator(self):
+        elf_table_name = intermed.IntermediateSymbolTable.create(self.context,
+                                                                 self.config_path,
+                                                                 "linux",
+                                                                 "elf",
+                                                                 class_types = elf.class_types)
+
+
+        vmlinux = self.context.modules[self.config['kernel']]
+        # These timekeeper variables are introduced in kernel 3.19
+        timekeeper = vmlinux.object_from_symbol(symbol_name = "tk_core").timekeeper
```

Comment:
This timekeeping code only supports a small range of kernels. For full support, all the variations would need to be added:

https://gitlab.research.volexity.com/Volcano/volatility_volc/-/blob/master/volatility/plugins/linux/common.py#L267

It would also make sense to put this handling in its own function so its easier to maintain and _generator does not get huge. I can help with this if needed/preferred.